vignettes/epidist.Rmd

```diff
@@ -1,201 +1,250 @@
 ---
-title: "Getting Started with Epidist"
-description: "A quick start guide to using the Epidist package"
-author: Epidist Team
+title: "Getting started with epidist"
+description: "A quick start guide to using the epidist R package"
 output: 
-  bookdown::html_vignette2:
+  bookdown::html_document2:
     fig_caption: yes
     code_folding: show
+    number_sections: true
 pkgdown:
   as_is: true
-csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+# csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
```

Comment:
?

Reply:
This was so that "as described in Park et al. ([2024](http://127.0.0.1:36043/rmd_output/0/#ref-park2024estimating)) and Charniga et al. ([2024](http://127.0.0.1:36043/rmd_output/0/#ref-charniga2024best))." appears rather than e.g. [1], [2].

Ideally perhaps the `.bibtex` format would also include linking to the paper. I don't know if any of the `citation-style-language/` ones allow you to do that?

Reply:
Got you. Happy to drop the csl then